vm-setup/roles/v1aX_integration_test/templates/cluster-template-workers.yaml

```diff
@@ -19,6 +19,7 @@ spec:
         cluster.x-k8s.io/cluster-name: ${ CLUSTER_NAME }
         nodepool: nodepool-0
     spec:
+      nodeDrainTimeout: ${ NODE_DRAIN_TIMEOUT }             
```

Comment:
Please do the same for KCP. (controlplane tempalte)

Reply:
Done